kratos/containers/pointer_vector_set.h

```diff
@@ -126,8 +126,8 @@ class PointerVectorSet final
     PointerVectorSet(TInputIteratorType First, TInputIteratorType Last, size_type NewMaxBufferSize = 1)
     : mSortedPartSize(size_type()), mMaxBufferSize(NewMaxBufferSize)
     {
-    for (; First != Last; ++First)
-        insert(begin(), *First);
+        mData.reserve(std::distance(First, Last));
```

Comment:
are u sure of this reserve? i would say that the reserve should be for the current size + this difference

Reply:
Since this is a ctor, the current size is always should be zero. Therefore, the reserve should be fine.

Reply:
true, sorry for the noise